pyteal/ast/acct.py

```diff
@@ -0,0 +1,62 @@
+from typing import TYPE_CHECKING
+
+from ..types import TealType, require_type
+from ..ir import Op
+from .expr import Expr
+from .maybe import MaybeValue
+
+if TYPE_CHECKING:
+    from ..compiler import CompileOptions
+
+
+class AcctParam:
+    @classmethod
+    def balance(cls, acct: Expr) -> MaybeValue:
+        """Get the current balance in microalgos an account.
+
+        Args:
+            acct: An index into Txn.accounts that corresponds to the account to check.
+                Must evaluate to uint64.
```

Comment:
I believe the account can be an integer index into the accounts array or the actual address of the account.

Reply:
I am not sure about this: see https://github.com/algorand/go-algorand/blame/feature/c2c/data/transactions/logic/TEAL_opcodes.md#L956

Reply:
the eval.go does call out to this method which just takes a `stackValue` and allows either an address or a ref to uint8 https://github.com/algorand/go-algorand/blob/4b1dfed080122c317df7e12e0d83c55871da7ff4/data/transactions/logic/eval.go#L3282

Reply:
my bad, it seems README is not quite up to dated.